test/tests/misc/check_error/tests

```diff
@@ -616,4 +616,55 @@
     input = 'missing_input_file.i'
     expect_err = "Unable to open file"
   [../]
+
+  [./kernel_with_aux_var]
+    type = 'RunException'
+    input = 'kernel_with_aux_var.i'
+    expect_err = "Cannot add Kernel for variable v, it is not a nonlinear variable"
+  [../]
+  [./bc_with_aux_var]
+    type = 'RunException'
+    input = 'bc_with_aux_var.i'
+    expect_err = "Cannot add BoundaryCondition for variable v, it is not a nonlinear variable"
+  [../]
+  [./aux_kernel_with_var]
+    type = 'RunException'
+    input = 'aux_kernel_with_var.i'
+    expect_err = "Cannot add AuxKernel for variable u, it is not an AuxVariable"
```

Comment:
probably `it is not an auxiliary variable`, like we spell out `nonlinear variable`.